src/senaite/core/registry/__init__.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+
+from plone.registry.interfaces import IRegistry
+from senaite.core.registry.schema import ISenaiteRegistry
+from zope.component import getUtility
+from zope.schema._bootstrapinterfaces import WrongType
+
+PREFIX = "senaite.core"
+
+
+def get_registry():
+    """Returns the registry utility
+
+    :returns: Registry object
+    """
+    return getUtility(IRegistry)
+
+
+def get(name, default=None):
+    """Get registry value by name
+    """
+    if not name.startswith(PREFIX):
+        name = "{}.{}".format(PREFIX, name)
```

Comment:
I suggest the add a parameter for the prefix, that defaults to `senaite.core` instead:

```
def get(name, prefix='senaite.core', default=None)
```

Otherwise, we won't be able to use this function for the retrieval of registry values from other add-ons, with a name that does not start with `senaite.core`, but with e.g `senaite.patient`

Reply:
This is by purpose. I would like to collect even the registry settings from patient in this global registry